README.md

```diff
@@ -56,10 +56,39 @@ $ source venv/bin/activate
 (venv)$ pip3 install -r requirements.txt
 ```
 
+## Developing Spam-Scanner
+
+The following commands are used to develop Spam-Scanner.
+
+### Front-end Development
+
+This uses webpack-dev-server to automatically apply any changes made to the front-end. Allowing you to see your changes in near-realtime from the browser.
+
+```bash
+# From root directory
+$ cd plino/web
+$ npm start
+# Front-end app then builds and runs from http://localhost:3000/
+```
+
+### Back-end development
+
+Run this command after making changes to the back-end to see them.
+
+```bash
+# From root directory
+$ make run
+# Back-end changes then runs on http://127.0.0.1:8000
+```
+
 ## Running Spam-Scanner
 
-The following command must be done to run Spam-Scanner
+The following commands must be executed to run Spam-Scanner with the python server serving the web page.
 
 ```bash
+# From root directory
+$ cd plino/web
```

Comment:
Instead of npm run-script build, use npm install
then npm start

useful if you don't have react to begin with